src/app/dashboard/_components/settings-screen/tabs/subscription/subscription-overview.tsx

```
@@ -0,0 +1,331 @@
+            console.log('Parsed status:', status);
+          } catch (parseError) {
+            console.error('Error parsing JSON response:', parseError);
+            throw new Error('Invalid JSON in subscription status response');
+          }
+        } catch (e) {
+          console.error('Error fetching subscription status:', e);
+          status = null;
+        }
+        // Map status to expected structure for AccountSubscriptionCard
+        let mappedSubscription = status;
+        if (status && status.plan_name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)